server/api/services/users.service.js

```diff
@@ -231,6 +234,53 @@ class UsersService {
       jwtSecret
     );
   }
+  /**
+   * Generate OTP for the user to reset password
+   * @param {String} roll - Roll number of the student
+   */
+   forgotPassword(roll) {
```

Comment:
Make this function asynchronous because we're using `await` within it. Similarly for the next function.